tests/benchmark/testdata/openapi1.yaml

```diff
@@ -0,0 +1,125 @@
+# Copyright 2021 Google LLC
```

Comment:
This file can be renamed to `openapi.yaml` now.

Reply:
done